Tests/Resources/configs/more_custom_size.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+    webtown_php_banner:
+        db_driver: orm
+        custom_size:
+            wide:
+                width: 100
+                height: 10
+            tall:
+                width: 10
+                height: 200
```

Comment:
A github tesz ide egy jelet. A fájloknak sortöréssel kell végződniük - ezt eddig nem mondtam. Javítsd kérlek mindenhol. Azt hiszem a PHPStormban is be lehet kapcsolni, hogy ezt automatikusan megtegye.